src/main/java/com/ericsson/ei/frontend/utils/BackEndInformationControllerUtils.java

```diff
@@ -60,8 +60,8 @@
         } catch (Exception e) {
             LOG.error("ERROR!\n" + e.getMessage());
             return new ResponseEntity<>(
-                    "[{\"name\":\"Unable to load instances\",\"host\":\"NO HOST\",\"port\":\"NO PORT\",\"contextPath\":\"/\"}]",
-                    getHeaders(), HttpStatus.OK);
```

Comment:
isn't '[' ']' required in a JSON object in some Web-Browsers and in JS Ajax response?. I remember we added those to solve the errors we got in some web-browsers.

Reply:
Not that I have seen. In this case though it was used a dummy false backend instance but it also never reached this part. I've been unsuccessful in even trying to get anything printed when the backend list is wrong. The application will just shutdown.